queue/queue.py

```diff
@@ -3,13 +3,17 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = []
```

Comment:
This would probably be better done with a linked list popping from the front will be super slow on an array.